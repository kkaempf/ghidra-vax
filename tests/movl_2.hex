d0 8f b2 04 14 20 51
