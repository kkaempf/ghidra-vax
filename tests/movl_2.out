0x00000000: MOVL I^#0x201404b2, R1
